### ISSUE TITLE
MSID: 88777 Version: 1 DOI: arXiv.2302.14157

### ISSUE THREAD

**Issue by nlisgo** [elifesciences/enhanced-preprints-import#1365]
"msas": "Neuroscience"
"msid": "88777"
"preprintDoi": "10.48550/arXiv.2302.14157"
"Reviewed Preprint posted": "2023-10-11"
"Sent for peer review": "2023-05-09"
"Posted to arXiv": "2023-02-27" (link: "Go to arXiv": "https://www.arxiv.org/abs/2302.14157v1/")

[PLACE PDF URL HERE WHEN AVAILABLE] See step 7

## Step 1. Inform bioRxiv

Who can help: @QueenKraken, @nlisgo, @scottaubrey

- [x] Manuscript is in data hub index (https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v2/by-publisher/elife/get-by-manuscript-id?manuscript_id=88777)

or (only one should be ticked. remove other from description.)

## Step 2. Create preview of manuscript

Who can help: @fred-atherden, @nlisgo, @scottaubrey

- [ ] Manuscript is available for preview (https://prod--epp.elifesciences.org/preview/88777/v1)
    - [ ] Trigger import for staging: https://prod--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://prod-elife-epp-data/data
    - [ ] Trigger import for staging: https://staging--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://staging-elife-epp-data/data
    - [ ] Pull request created to [enhanced-preprint-data](https://github.com/elifesciences/enhanced-preprints-data/pulls)
    - [ ] Post link to pull request below

Pull request: [PLACE LINK TO PULL REQUEST HERE]

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-manuscript

## Step 3: Awaiting public reviews and QC

Who can help: Production team

- [ ] QC passed or QC not needed

Who can help: Editorial team

- [ ] Public reviews available on sciety (https://sciety.org/articles/activity/10.48550/arXiv.2302.14157)
- [x] Manuscript and editor details at the top of this issue (Supplied by Editorial team)
- [x] Reviews returned by EPP - https://prod--epp.elifesciences.org/api/reviewed-preprints/88777/v1/reviews

<details>
<summary>Example</summary>

```
"msas": "Genetics and Genomics", "Neuroscience"
"msid": "84628"
"version": "1"
"preprintDoi": "10.1101/2022.10.28.514241"
"articleType": "Reviewed Preprint"
"status": "Published from the original preprint after peer review and assessment by eLife."

"Reviewed Preprint posted": "2023-01-02"
"Sent for peer review": "2022-10-28"
"Posted to bioRxiv": "2022-11-21" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2022.10.28.514241v1")

Editors:
Reviewing Editor
Michael B Eisen
University of California, Berkeley, United States
Senior Editor
Michael B Eisen
University of California, Berkeley, United States
```

</details>

## Step 4: Modify manuscripts.json (no PDF)

- [ ] Reviewed preprint is published to EPP (https://elifesciences.org/reviewed-preprints/88777)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below
- [ ] Request that a doi be registered
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

Pull request: [PLACE LINK TO PULL REQUEST HERE]
[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

<details>
<summary>Instructions to modify manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/publish-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/334/files

Once the pull request is merged in it should be available a few minutes later.

</details>

### Request that a doi

Post the following in [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77):

```
@Fred can you register a doi for https://elifesciences.org/reviewed-preprints/88777
```

## Step 5: Awaiting search reindex

- [ ] Reviewed preprint is avaliable on journal homepage (https://elifesciences.org)

The search reindex is triggered once an hour. We need the reviewed preprint to be indexed as the search application serves the journal homepage.

<details>
<summary>Additional info</summary>

If needed, the jenkins pipeline to reindex search can be triggered sooner.

https://alfred.elifesciences.org/job/process/job/process-reindex-reviewed-preprints/

</details>

## Step 6: Published! Request PDF generation

- [ ] Post the link to the PDF url at the top of the issue
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

Post the following to the [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) on slack:

```
@Ryan Dix-Peek please can you generate a PDF for https://elifesciences.org/reviewed-preprints/88777
```

## Step 7: Introduce PDF to data folder and git repo

- [ ] PDF is avaliable at https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/88777/v1/88777-v1.pdf

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-pdf

## Step 8: Add PDF url to manuscripts.json

- [ ] Reviewed preprint PDF is available for download (https://prod--epp.elifesciences.org/reviewed-preprints/88777)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below

[PLACE LINK TO PULL REQUEST HERE]

<details>
<summary>Instructions to add PDF url to manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/add-pdf-url-to-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/397/files

Once the pull request is merged in it should be available a few minutes later.

</details>

## Step 9: Done!

- [ ] Kettle is on!

**Comment by fred-atherden** [comment]
Some issue with loading the package at `s3://prod-elife-epp-meca/88777-v1-meca.zip`. 

Log message is
```
getting image paths from /var/folders/q3/y4w9ffqd3x7_442hq9wd3lnw0000gp/T/tmp.i1bJs1WZ/manifest.xml ...
cp: /var/folders/q3/y4w9ffqd3x7_442hq9wd3lnw0000gp/T/tmp.i1bJs1WZ/2302.14157/2302.14157v3_eqn1.jpg: No such file or directory

```

**Comment by fred-atherden** [comment]
Above is now fixed

**Comment by fred-atherden** [comment]
- Chinese representation of author names are captured as suffix, which is semantically incorrect but means they actually display. I'm inclined to leave it as is. 
- Equation labels not within images, but these are captured (and not rendered by EPP), so consider this a platform issue which we can proceed with. 


The analytical results section contains display equations that are treated as figures and he preceding text has unfortunately been parsed as the caption for those equations/figures:

![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/3103a63c-ec10-467c-ae06-70f5a5026721)

This needs fixing with a workaround. 

Figure captions are all captured as titles.

Some of the equations are not quite right even though they are images:
e.g. EPP:


![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/f0eb39be-f1d5-44dd-987e-a8ff635f926b)




PDF:


![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/942173ed-dcde-49e8-909f-4235817f2092)

**Comment by fred-atherden** [comment]
I started trying to correct these myself but there are so many that are wrong when compared with the PDF. @JGilbert-eLife please could you go back to Exeter about this one?

Here are the issues:

1. Figure captions. First sentence should be title, rest should be paragraph. 

2. There are loads of equations in the 'Proof of Theorems' sections that are just wrong for some reason (see example above). 

3. Also while we're asking exeter to make these changes, please can you ask that they include the equation numbers in the images?

4. There's at least one equation captured inline which would be better served as a display equation:
EPP:



![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/ef4491d4-0d8d-42ba-9e35-5a0cd44ebe39)


PDF:



![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/16e6ce3f-9527-4470-817c-76b9e2f48da0)

5. This ref can be captured more appropriately:


![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/f9ea6063-52e5-46d2-95fa-41e341fe4c82)



```xml
<ref id="c49"><label>49.</label><mixed-citation publication-type="other"><person-group person-group-type="author"><string-name><given-names>Tanguy</given-names> <surname>Fardet</surname></string-name>, <string-name><given-names>Rajalekshmi</given-names> <surname>Deepu</surname></string-name>, <string-name><given-names>Jessica</given-names> <surname>Mitchell</surname></string-name>, <string-name><given-names>Jochen Martin</given-names> <surname>Eppler</surname></string-name>, <string-name><given-names>Sebastian</given-names> <surname>Spreizer</surname></string-name>, <string-name><given-names>Jan</given-names> <surname>Hahne</surname></string-name>, <string-name><given-names>Itaru</given-names> <surname>Kitayama</surname></string-name>, <string-name><given-names>Piotr</given-names> <surname>Kubaj</surname></string-name>, <string-name><given-names>Jakob</given-names> <surname>Jordan</surname></string-name>, <string-name><given-names>Abigail</given-names> <surname>Morrison</surname></string-name></person-group>, <etal>et al.</etal> <article-title>Nest 2.20. 1. Technical report, Computational and Systems Neuroscience</article-title>, <year>2020</year>.</mixed-citation></ref>
```

**Comment by JGilbert-eLife** [comment]
Sure - I assume the initial package was prepared before I took over handling these, as I can't find a thread for it in the inbox. I will ask them to do it again - did you need me to tell them about the suffix issue you mentioned as well?

**Comment by fred-atherden** [comment]
Yeah it's probably one I requested before you took over (likely why there are so many issues!). if you search using the preprint DOI the email might come up. 

Good question. Maybe we should ask them to capture it correctly using [`name-alternatives`](https://jats.nlm.nih.gov/archiving/tag-library/1.3/element/name-alternatives.html), and then I can add an xsl to fudge the XML consumed by EPP so they display as they do currently (but the package itself is correct). WDYT?

**Comment by JGilbert-eLife** [comment]
Yeah, sounds good to me!

And yep, got it based off the DOI.

**Comment by fred-atherden** [comment]
Great - thanks!

I think it should be a conference proceeding (although either would be fine I think if you think otherwise). The major problematic issue I think is this:
```xml
<article-title>Nest 2.20. 1. Technical report, Computational and Systems Neuroscience</article-title>
```

i.e. no `<source>` or equivalent.

**Comment by JGilbert-eLife** [comment]
OK - I'll ask them to make the change to a proceeding

**Comment by fred-atherden** [comment]
@JGilbert-eLife I've loaded this one - but the quality of the images looks terrible. Please could you check the original package (and if they're bad there too, ask Exeter to fix)?

**Comment by fred-atherden** [comment]
Thanks!

**Comment by JGilbert-eLife** [comment]
Meca package reloaded with better images.

**Comment by fred-atherden** [comment]
Images look better - Encoda has slapped some captions on display equations as if they were figures though, so this needs fixing.
